scrapi/base/schemas.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import unicode_literals
+
+from copy import deepcopy
+from dateutil.parser import parse
+
+from nameparser import HumanName
+
+
+def default_name_parser(names):
+    contributor_list = []
+    for person in names:
+        name = HumanName(person)
+        contributor = {
+            'prefix': name.title,
+            'given': name.first,
+            'middle': name.middle,
+            'family': name.last,
+            'suffix': name.suffix,
+            'email': '',
+            'ORCID': ''
+        }
+        contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def format_tags(all_tags):
```

Comment:
I'd modify this function to more match something like what we used to use in the old class based harvester for dealing with tags - instead of splitting on spaces (which would remove meaning from lots of the multi-word tags) you could split on commas, something like this: 

```
    def format_tags(all_tags):
        for tag in all_tags:
            if ', ' in tag:
                tags.remove(tag)
                tags += tag.split(',')

        return [util.copy_to_unicode(tag.lower().strip()) for tag in tags]
```